install-apps.sh

```diff
@@ -0,0 +1,3 @@
+#!bin/bash
```

Comment:
It'd be even better if you use env script:
```suggestion
#! /usr/bin/env bash
```